.travis.yml

```diff
@@ -1,2 +1,8 @@
-language: c
-script: bin/shpec
+language: sh
+script:
+  - time bash bin/shpec -f 1 shpec/shpec_shpec.sh
+  - time dash bin/shpec -f 1 shpec/shpec_shpec.sh
+  - time ksh  bin/shpec -f 1 shpec/shpec_shpec.sh
```

Comment:
Any specific reason we are not testing against zsh?

(if it's about the `end` keyword, it can be disabled, cf https://github.com/AdrieanKhisbe/shpec/blob/dev/shpec.plugin.zsh#L15)